tests/functional/cylc-install/00-simple.t

```diff
@@ -59,7 +59,7 @@ __ERR__
 TEST_NAME="${TEST_NAME_BASE}-REG-install-ok"
 run_ok "${TEST_NAME}" cylc install "${RND_SUITE_NAME}"
 contains_ok "${TEST_NAME}.stdout" <<__OUT__
-INSTALLED $RND_SUITE_NAME from ${RND_SUITE_SOURCE} -> ${RND_SUITE_RUNDIR}/run1
+INSTALLED $RND_SUITE_NAME/run1 from ${RND_SUITE_SOURCE} -> ${RND_SUITE_RUNDIR}/run1
```

Comment:
If the first and last part of this message are always the same (?) could we change it to:

```suggestion
INSTALLED: ${RND_SUITE_SOURCE} -> ${RND_SUITE_RUNDIR}/run1
```

Reply:
They are slightly different (probably not different enough to warrant not accepting your suggestion).... first one is the named workflow i.e what users will use to interact... e.g.  `flow_name/run1`
The last one is the directory in which it is installed to, which does indeed end in the named_workflow.
Happy to accept your change if you confirm you are happy with that.

Reply:
I think `$RND_SUITE_NAME/run1` is more useful than `${RND_SUITE_RUNDIR}/run1`.
How about:
```
INSTALLED $RND_SUITE_NAME/run1 from ${RND_SUITE_SOURCE}
```
(also, this is shorter!)